src/gwt/src/org/rstudio/studio/client/workbench/prefs/views/EditingPreferencesPane.java

```diff
@@ -77,6 +77,7 @@ public EditingPreferencesPane(UserPrefs prefs,
             "When enabled, the indentation for documents not part of an RStudio project " +
             "will be automatically detected."));
       editingPanel.add(checkboxPref("Insert matching parens/quotes", prefs_.insertMatching()));
+      editingPanel.add(checkboxPref("Rainbow parentheses", prefs_.rainbowParentheses()));
```

Comment:
Does everything in this pane still fit (vertically)? Noticed this pane is getting very full. I ran into a problem adding a checkbox to the Code / Display pane in #7030 and had to remove some other spacing or it was slightly clipped at the bottom

Reply:
Makes me think it might be time to support scrolling preferences panes...

Reply:
It does, there's still room for like...one more checkbox. It's definitely cramped though. We are in the situation where we want to start thinking about a different format for our options panel.